### ISSUE TITLE
Support for Vite 5?

### ISSUE THREAD

**Issue by WonderPanda** [mathe42/vite-plugin-comlink#128]
Hey there! Thanks for the cool library. I've been playing around trying to see if I can get this working in a newer app using Vite version 5.0.12.

Following the instructions from the readme, I end up with an error

```
WorkerPage.tsx?t=1707407479840:14 Uncaught ReferenceError: ComlinkWorker is not defined`
```

It seems like the types for ComlinkWorker are coming from the types added to `vite-env.d.ts` but they are not actually available.

Has anyone had success getting this to work?

**Comment by WonderPanda** [comment]
Yeah I can follow the type reference. Everything looks like its getting type checked correctly in the IDE but it fails at runtime with the reference error I linked above :thinking:

**Comment by mathe42** [comment]
Please share a minimal reproduction. It looks to me as you didn't register the plugin correctly.
